rev_news/drafts/edition-15.md

```diff
@@ -248,6 +248,7 @@ which is very useful when rebasing a large patch series.  I also have to mention
 
 __Various__
 
+* BitKeeper, the inspiration for Git and Mercurial, [was released as Open Source](https://users.bitkeeper.org/t/bk-7-2ce-released-2016-05-09/93) (see also the discussion on [Hacker News](https://news.ycombinator.com/item?id=11667494) as well as [Linux Weekly News](https://lwn.net/Articles/686896/)).
```

Comment:
The [LWN news](https://lwn.net/Articles/686896/) starts with "BitKeeper, the inspiration behind Git and Mercurial" too.

And as you suggest in the second paragraph, maybe it is ok with the meaning: "the inspiration to create a new DVCS".